src/components/App/App.js

```diff
@@ -2,30 +2,35 @@ import React, { Component } from 'react'
 import AllMovies from '../AllMovies/AllMovies'
 import MovieInfo from '../MovieInfo/MovieInfo'
 import './App.css'
-import movieData from '../../movieData'
 
 class App extends Component {
   constructor() {
-    super()
+    super();
     this.state = {
-      movies: movieData.movies,
-      currentCard: null
+      movies: [],
+      currentMovie: null
     }
   }
 
   handleClick = (event) => {
-    event.preventDefault();
-    this.setState({ currentCard: Number(event.target.id) })
+    fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${Number(event.target.id)}`)
```

Comment:
Good stuff! Just don't forget that API calls should be moved into their own file, per the rubric.